pipeline_steps/common.groovy

```diff
@@ -278,13 +278,54 @@ def archive_artifacts(){
   } finally{
     // still worth trying to archiveArtifacts even if some part of
     // artifact collection failed.
-    archiveArtifacts(
-      allowEmptyArchive: true,
-      artifacts: "artifacts_${env.BUILD_TAG}.tar.bz2,artifact_index.txt"
-    )
+    stash(
+      name: "artifacts",
+      includes: "artifacts_${env.BUILD_TAG}.tar.bz2")
+    node(){
+      sh "rm -f index.html"
```

Comment:
My thought process for this was that the playbook would do just uploading and getting the file to the proper location would be done outside of that in case we want to reuse the playbook in a different way. I'll update the playbook to make the destination a parameter, though.